dirtree.go

```diff
@@ -4,14 +4,33 @@ import (
 	"fmt"
 	"os"
 	"path/filepath"
+	"strings"
 
 	gitignore "github.com/sabhiram/go-gitignore"
 )
 
 func main() {
-	rootDir := "."
+	var rootDir string
+
+	switch len(os.Args) {
+	case 1:
+		rootDir = "."
+		fmt.Println(".")
+	case 2:
+		rootDir = strings.Join(os.Args[1:2], "")
```

Comment:
In case of 2 arguments, we can directly access the second element.
```suggestion
		rootDir = os.Args[1]
```